From 046dc697f49f13443cd9c4f7ddc76ba277c799d4 Mon Sep 17 00:00:00 2001
From: Jean-Christophe Fillion-Robin <jchris.fillionr@kitware.com>
Date: Mon, 6 Nov 2017 21:44:29 -0500
Subject: [PATCH 1/3] Prevent incorrect include of "io.h" found in libmpdec
 directory

This commit improves support for building python with built-in extensions
on windows where the header provided by libmpdec would be included instead
of the system one.
---
 Modules/_decimal/libmpdec/io.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Modules/_decimal/libmpdec/io.c b/Modules/_decimal/libmpdec/io.c
index e7bd6ae..ecebe82 100644
--- a/Modules/_decimal/libmpdec/io.c
+++ b/Modules/_decimal/libmpdec/io.c
@@ -37,7 +37,7 @@
 #include <stdlib.h>
 #include <string.h>
 
-#include "io.h"
+#include "mpd_io.h"
 #include "typearith.h"
 
 
-- 
2.47.2

